recipe/build.sh

```diff
@@ -1,25 +1,19 @@
 mkdir build
 cd build
 
-if [[ "$target_platform" == osx* ]]; then
-  LLDB_USE_SYSTEM_DEBUGSERVER=ON
-else
-  LLDB_USE_SYSTEM_DEBUGSERVER=OFF
-fi
```

Comment:
Why change this? There's no system debugserver on linux

Reply:
I can change it back, I assumed it would just be a no-op when not building on mac.